DQM/TrackingMonitorSource/interface/TrackMultiplicityFilter.h

```diff
@@ -0,0 +1,39 @@
+#ifndef TRACKMULTIPLICITYFILTER_H
+#define TRACKMULTIPLICITYFILTER_H
```

Comment:
this doesn't comply with rule 4.1 [1] (that you follow for all the other plugins in this package).
Even better this could be merged with the plugin implementation itself, see rule 6.7 [1]

[1] https://cms-sw.github.io/cms_coding_rules.html